modules/youtube.py

```diff
@@ -0,0 +1,18 @@
+from .base import Module
+import requests
+from bs4 import BeautifulSoup
```

Comment:
Hmmm... I love a good BeautifulSoup hack just as much as the next guy or girl, but YouTube has a pretty comprehensive API which might be preferable to this. Maybe try something similar to this? https://developers.google.com/youtube/v3/code_samples/python#search_by_keyword